server/src/main/scala/org/http4s/server/middleware/CSRF.scala

```diff
@@ -446,7 +446,6 @@ object CSRF {
   ///
 
   val SigningAlgo: String = "HmacSHA1"
-  val SHA1ByteLen: Int = 20
```

Comment:
If we deprecate this value, we could release this as a fix on series/0.20.

Reply:
I think we should (deprecate it). Do I need to do anything on my side to do that?

Reply:
I just went ahead and did it, because it's easier than taking a reviewed PR and moving it to a different base.  But deprecating a value [looks like this](https://github.com/http4s/http4s/blob/series/0.20/server/src/main/scala/org/http4s/server/middleware/CSRF.scala#L449-L450).